src/nvim/api/private/dispatch.h

```diff
@@ -4,6 +4,7 @@
 #include <stdint.h>
 
 #include "nvim/api/private/defs.h"  // IWYU pragma: keep
+#include "nvim/errors.h"  // NOLINT(build/include_defs)
```

Comment:
Should it be errors_defs.h? This NOLINT seems to be ignored by clint. 

Without this include, linker fails:

```
ack-protector-strong  -lm  -lutil  .../neovim/.deps/usr/lib/libuv.a  -ldl && :
Undefined symbols for architecture x86_64:
  "_bot_top_msg", referenced from:
      _searchit in search.c.o
      _search_for_exact_line in search.c.o
      _spell_move_to in spell.c.o
  "_e_api_error", referenced from:
      _api_wrapper in funcs.c.o
  "_e_api_spawn_failed", referenced from:
      _f_rpcstart in funcs.c.o
  "_e_argreq", referenced from:
      _do_augroup in autocmd.c.o
      _do_write in ex_cmds.c.o
      _do_one_cmd in ex_docmd.c.o
...
```

Reply:
Include in in main.c